LICENSE

```diff
@@ -0,0 +1,674 @@
+                    GNU GENERAL PUBLIC LICENSE
```

Comment:
I suggest we use MIT instead of GPL to enable commercial use more widely (even though it does not affect us)

Reply:
Switched in commit 602527ef414788626cabf23817ce9bd4dd17b8b7.